sru/modules.py

```diff
@@ -27,6 +23,11 @@ class SRUCell(nn.Module):
     scale_x: Tensor
     weight_proj: Optional[Tensor]
 
+    initialized = False
+    elementwise_recurrence_inference = None
```

Comment:
Please add a note about this function initialization to the SRUCell docstring *or* the docstring for sru/modules.py (it lacks a module docstring now, it probably should have one)

Reply:
Aded